packages/svelte/src/internal/client/runtime.js

```diff
@@ -1548,11 +1484,9 @@ export function prop_source(props_obj, key, immutable, default_value, call_defau
 				return;
 			}
 
-			if (not_equal(immutable, propagating_value, possible_signal.v)) {
```

Comment:
Why can we remove this? I'm a bit surprised that no tests fail because removing this seems unrelated. Is it because the parent now always decides what to do with the update?